internal/cmd/config/config_test.go

```diff
@@ -116,6 +116,20 @@ func TestDevController(t *testing.T) {
 		assert.Equal(t, []string{"*"}, l0.CorsAllowedOrigins)
 		assert.Nil(t, l0.CorsDisableDefaultAllowedOriginValues)
 
+		// Implicitly with explicit wildcard
```

Comment:
It might be worthwhile adding a test to verify the implicit `cors_enabled` with explicit non wildcard `cors_allowed_origins` scenario, then ensure `desktopCorsOrigin` is appended to the `cors_allowed_origins` list as expected.

The only test of the appending functionality is when `cors_allowed_origins` is not specified

Reply:
Done!